src/SSHLibrary/library.py

```diff
@@ -11,12 +11,13 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
+from __future__ import print_function
```

Comment:
Do we really need this `__future__` import for only one `print` statement?

Reply:
I think it's fine. Better to use print as a function everywhere. It probably also makes IDEs inspecting Python 2 and 3 compatibility happier.